docs/src/docs/pykitops.md

```diff
@@ -0,0 +1,5 @@
+# KitOps Python Library
+
+This site contains the project documentation for the PyKitOps SDK project - a python library for working with [KitOps' ModelKits](https://kitops.ml) in code. You can find [KitOps and ModelKit documentation](https://kitops.ml/docs/overview.html) on the KitOps site.
```

Comment:
Since these docs will be hosted elsewhere we should include a link back to the PyKitOps github repo. We can say something like "The KitOps Python library is an open source project hosted on GitHub [link]"

Reply:
@bmicklea done, can you re-check please?